src/ui/pages/BridgeForm/BridgeForm.tsx

```diff
@@ -356,15 +360,23 @@ function BridgeFormComponent() {
   const formState = useMemo(
     () => ({
       ...defaultFormValues,
+      to: null,
```

Comment:
why is this needed if it's null anyway?

Reply:
The type of the field is `to: string | null`.
Because the `to` field can not be "undefined" – we either want to send to another address or not, there is no 3rd option

Reply:
Got it! It's `string | undefined` though in SendForm